### PR TITLE
fix documentation links

### PR DESCRIPTION
**Pull request by bparees**
related to https://github.com/openshift/origin/issues/16121

### CODE REVIEW

**Comment by bparees** [comment]
@hhorak ptal

**Comment by pkubatrh** [comment]
Looking at the linked Openshift issue I see not working README links (pointing to non-clickable symlinks) being a problem. Postgresql has the links pointing to the actual README files so there should be no such problem here.

The only unusable links now are the ones poinitng to EOL images (9.2, 9.4) as we do not maintain their documentation anymore. If that is something that needs fixing from your point of view please change the PR so that only those links are changed.

**Comment by praiskup** [comment]
Agreed with @pkubatrh , btw. what would be the right time to drop the EOL images from templates/imagestreams?

File: imagestreams/postgresql-centos7.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         "annotations": {
           "openshift.io/display-name": "PostgreSQL 9.4",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 9.4 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/tree/master/9.4/README.md.",
+          "description": "Provides a PostgreSQL 9.4 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
```
[the remark this answers is not in the document]
Comment (bparees) [reply imagestreams/postgresql-centos7.json]: that link (`https://github.com/sclorg/postgresql-container/blob/master/README.md`) works fine for me?

**Comment by bparees** [comment]
> The only unusable links now are the ones poinitng to EOL images (9.2, 9.4) as we do not maintain their documentation anymore. If that is something that needs fixing from your point of view please change the PR so that only those links are changed.

Rather than continue to allow things to bit-rot as you drop them, i'd rather point to this repo and let people dig into to find the docs for their specific version (and also be made aware of other versions being available).

Since the original issue (https://github.com/openshift/origin/issues/16121) was not addressed for almost a year after it was reported, I would prefer a solution that isn't going to require on-going maintenance to keep it valid.

> Agreed with @pkubatrh , btw. what would be the right time to drop the EOL images from templates/imagestreams?

The templates should not point to EOL images, they should point to the current latest imagestreamtag version (but not the "latest" tag itself), typically.  And typically be parameterized.

The imagestreams can never be dropped because people can have resources that still point to the old imagestreamtags and if you drop them, those references will break.

File: imagestreams/postgresql-centos7.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (omron93) [reply imagestreams/postgresql-centos7.json]: Right. Sorry, I've copied the wrong one :-(

**Comment by praiskup** [comment]
Thanks, looks fine to me.

**Pull request merged**